### ISSUE TITLE
I as a sales person,  I want to generate daily sales report

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa#41]

**Comment by lcabrini** [comment]
This looks good (after a bit of editing) and is required functionality. I propose that we move this to the product backlog.

**Comment by BorderITS** [comment]
That's true because it's part of this sprint

**Comment by lcabrini** [comment]
So, do you agree that we move it to the product backlog? (Y/N)

**Comment by BorderITS** [comment]
Yes which I  have done

**Comment by lcabrini** [comment]
Yes that is fine, but can we agree first and then move? Otherwise we defeat the purpose of proposing and agreeing.

**Comment by BorderITS** [comment]
Okay sorry for that

**Comment by lcabrini** [comment]
By the way, there is still an issue with this. It belongs to the UI project as well as the user stories project. Which is it? It looks like a user story and not a UI task. Moreover, it is placed in design proposed, so where is the proposed design? We can't have something with a proposed design if there is no mockup for us to look at.

**Comment by BorderITS** [comment]
ok then should take that also form design proposed ?

**Comment by lcabrini** [comment]
1. This is a user story. it does not belong in the UI project at all. Any issue should belong in exactly one project. Either it is a user story, a UI design task or a development task. Never more than one.

2. A UI task can only be put in the Design Proposed column if a design is proposed. That would normally be done by the designer. If there is no mockup available, then we can safely assume that task is still in need of a design.

**Comment by BorderITS** [comment]
Ok I have removed it from the UI

**Comment by lcabrini** [comment]
Good, now if we can apply the Sales label and add it to our milestone, I propose that we move this to product backlog.

**Comment by BorderITS** [comment]
Ok agreed !

**Comment by BorderITS** [comment]
you can move it now

**Comment by lcabrini** [comment]
Well, it was already in the product backlog. But it was missing label and milestone. Can we both try to make sure that before we move anything into product backlog that these things have been applied. 

I have done it to this issue.
